packages/extension/src/languages/en.json

```diff
@@ -119,6 +119,25 @@
   "pages.register.connect-ledger.use-hid-confirm-title": "Unable to use Web HID",
   "pages.register.connect-ledger.use-hid-confirm-paragraph": "Please enable ‘experimental web platform features’ to use Web HID",
 
+  "pages.register.connect-keystone.title": "Please Connect Your Hardware Wallet",
+  "pages.register.connect-keystone.step-text": "Step {num}",
+  "pages.register.connect-keystone.step-1": "Tap “<span>Connect Software Wallet</span>” at the bottom left corner on the Keystone device.",
+  "pages.register.connect-keystone.step-2": "Select “<span>Keplr Wallet</span>”.",
+  "pages.register.connect-keystone.step-3": "Click on the “<span>Sync Keystone</span>” button below to scan the QR code displayed on the Keystone device.",
+  "pages.register.connect-keystone.click-to-view-tutorial": "Click here to view detailed tutorial",
+  "pages.register.connect-keystone.sync": "Sync Keystone",
+  "pages.register.connect-keystone.title-scan": "Scan the QR Code",
+  "pages.register.connect-keystone.paragraph-scan": "Scan the QR code displayed on your Keystone device",
+  "pages.register.connect-keystone.position-qrcode": "Position the QR code in front of your camera.",
+  "pages.register.connect-keystone.no-camera-permission": "No camera permission",
+  "pages.register.connect-keystone.enable-camera": "Please enable your camera permission via [Settings]",
+  "pages.register.connect-keystone.invalid-qrcode": "Invalid QR code",
+  "pages.register.connect-keystone.select-valid-qrcode": "Please ensure you have selected a valid QR code from your Keystone device.",
+  "pages.register.connect-keystone.tutorial": "Tutorial",
+  "pages.register.connect-keystone.note-suggest-chain": "Please note that when adding specific chains to your wallet, the ‘Suggest Chain’ feature may not work.",
```

Comment:
@renfengshi Could you change to `Non-Native Chain` from ‘Suggest Chain’
We didn't use ‘Suggest Chain’ anymore